modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/EditObjectValidation.tsx

```diff
@@ -93,15 +93,17 @@ export default function EditObjectValidation({
 		}
 		catch (error) {
 			const {detail, message} = error as ErrorDetails;
-			const {fieldName, message: detailMessage} = JSON.parse(
-				detail as string
-			) as {
-				fieldName: keyof ObjectValidationErrors;
-				message: string;
-			};
 
-			setErrorMessage({[fieldName]: detailMessage});
+			if (detail) {
+				const {fieldName, message: detailMessage} = JSON.parse(
+					detail as string
+				) as {
+					fieldName: keyof ObjectValidationErrors;
+					message: string;
+				};
 
+				setErrorMessage({[fieldName]: detailMessage});
+			}
 			openToast({message, type: 'danger'});
```

Comment:
Hey @pedro-oliveira446, could you please add a blank line here?

Reply:
Thx Dan